src/app/routes/cpsAsset/getInitialData/processMostWatched/index.js

```diff
@@ -0,0 +1,70 @@
+import pathOr from 'ramda/src/pathOr';
+import nodeLogger from '#lib/logger.node';
+import { MOST_WATCHED_PROCESS_ERROR } from '#lib/logger.const';
+import filterPopularStaleData from '#app/lib/utilities/filterPopularStaleData';
+
+const logger = nodeLogger(__filename);
+
+const processMostWatched = ({ data, path, service, toggles }) => {
+  if (!data) {
+    return data;
+  }
+
+  const { mostWatched } = data;
+
+  if (!mostWatched) {
+    return data;
+  }
+
+  const defaultToggle = {
+    enabled: false,
+    value: `{"numberOfItems": 10}`,
+  };
+
+  const { mostWatched: mostWatchedToggle } = toggles;
+  if (!mostWatchedToggle) {
+    logger.warn(MOST_WATCHED_PROCESS_ERROR, {
+      message: 'Invalid most watched toggle',
+      service,
+      path,
+    });
+  }
```

Comment:
Do we need this - there's a log of toggling within the toggles fetch itself, so maybe this is not required here.

Reply:
We could have an improperly configured value